packages/p2p/src/components/nickname/nickname-form.jsx

```diff
@@ -66,7 +66,11 @@ const NicknameForm = ({ handleClose }) => {
                     </div>
                 </div>
             )}
-            <Formik validate={validatePopup} initialValues={{ nickname: '' }} onSubmit={handleSubmit}>
+            <Formik
+                validate={validatePopup}
+                initialValues={{ nickname: '' }}
+                onSubmit={async values => handleSubmit(values)}
```

Comment:
If we don't have `await` here, the code execution wouldn't be paused (while we're supposed to wait for the response to be our resolved promise) so I'm not sure if this actually fixes the issue.